packages/wp-story-editor/src/components/documentPane/publish/publish.js

```diff
@@ -318,8 +318,8 @@ function PublishPanel({ nameOverride }) {
                     node.focus();
                   }
                 }}
-                width={72}
-                height={96}
+                $width={72}
+                $height={96}
```

Comment:
These were being passed into the media input component (And eventually the button) but shouldn't be. They're applied in the `StyledMedia` wrapper